open-reac/src/main/java/com/powsybl/openreac/parameters/input/json/VoltageLimitOverrideDeserializer.java

```diff
@@ -26,23 +26,33 @@ public VoltageLimitOverrideDeserializer() {
 
     @Override
     public VoltageLimitOverride deserialize(JsonParser parser, DeserializationContext deserializationContext) throws IOException {
-        double deltaLowVoltageLimit = 0;
-        double deltaHighVoltageLimit = 0;
+        String voltageLevelId = null;
+        VoltageLimitOverride.VoltageLimitType type = null;
+        boolean isRelative = true;
+        double overrideValue = 0;
```

Comment:
You might want to initialize to Double.NaN and then the check that the value is in the file will be done when calling the constructor.

Reply:
Yes, but @p-arvy value must be defined ad so checked at construction.